sktime/performance_metrics/forecasting/probabilistic/tests/test_distr_metrics.py

```diff
@@ -4,14 +4,18 @@
 import pandas as pd
 import pytest
 
-from sktime.performance_metrics.forecasting.probabilistic._classes import CRPS, LogLoss
+from sktime.performance_metrics.forecasting.probabilistic._classes import (
+    CRPS,
+    AUCalibration,
+    LogLoss,
+)
 from sktime.proba.normal import Normal
 from sktime.proba.tfp import TFNormal
 from sktime.utils.dependencies import _check_soft_dependencies
 
 warnings.filterwarnings("ignore", category=FutureWarning)
 
-DISTR_METRICS = [CRPS, LogLoss]
```

Comment:
Could we automate it using the tag system? -> Would be a new issue.

Reply:
yes, it's automated in `skpro`, but `sktime` has never been updated to this.

It would be resolved if we move the metrics out into that dependency, but that will take longer. We can just copy code, shorter-term and then deduplicate.